### DISCUSSION TITLE
제가 만든 월드에서 제가 만든 다른 월드로 이동시키고 싶습니다.

### DISCUSSION THREAD

**Discussion by hhnmsLeesan** [naverz/zepeto-studio-kor#480] [5. Q&A]
안녕하세요!

제가 구상하고 있는 월드의 사이즈가 커서 500Mb에 맞추기 어려울 것 같아 두 개의 월드를 만든 뒤 월드간에 포탈 등을 통해 이동할 수 있게 하고자 합니다. 이것이 제페토 가이드에 있는 URL Scheme를 활용하면 충분히 가능할 것 같은데요.
(https://studio.zepeto.me/kr/guides/using-urlscheme)
혹시 이런 식으로 제가 만든 월드끼리 연결을 해둘 경우 '외부 링크를 통해 연결을 시도하면 안된다' 라는 심사 가이드라인에 저촉되는지가 궁금합니다. 

감사합니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요.

현재는 다른 월드로의 이동 기능을 제공하지 않고 있습니다.

이후 버전에서는 논의될 수도 있는 사항이지만 정확한 업데이트 시점을 공유해 드릴 수는 없는 점 양해 부탁드립니다.
